test_conformance/subgroups/subhelpers.h

```diff
@@ -1098,8 +1239,8 @@ struct test
         Ty mapout[LSIZE];
         std::stringstream kernel_sstr;
 
-        kernel_sstr << "#define NON_UNIFORM " + std::to_string(NON_UNIFORM)
-                + " \n";
+        kernel_sstr << "#define NON_UNIFORM_WG_SIZE "
+                + std::to_string(NON_UNIFORM_WG_SIZE) + " \n";
```

Comment:
I don't understand. kernel_sstr is a stringstream - shouldn't these "+"s be "<<" instead?
It seems to me you're generating temp strings (from the  '+') unnecessarily.